portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,19 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addGreeting() {
+  const greetingPromise = fetch("/data");
+  greetingPromise.then(handleGreeting);
+  
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function handleGreeting(response) {
+  const greetingTextPromise = response.text();
+  greetingTextPromise.then(displayGreeting);
+}
 
-  // Add it to the page.
+function displayGreeting(greetingText) {
   const greetingContainer = document.getElementById('greeting-container');
```

Comment:
same comment here

Reply:
Addressed in new commit https://github.com/malcolmrjones/step/pull/4/commits/212ec810f2b3c85352ec4ffa2b0384167a53c388